css-content/Overview.bs

```diff
@@ -601,7 +601,8 @@ The ''leader()'' function</h4>
 	</dl>
 
 	<pre class="prod">
-		<dfn><<leader-type>></dfn> = leader( dotted | solid | space | <<string>>);
+		<dfn><<leader()>></dfn> = leader( <<leader-type>> )
```

Comment:
This should be defined as just `<dfn>leader()</dfn>`. (The grammar non-terminal automagically works when the function is defined normally.)